int16[6] adsGain
float32 vref
float32[6] adsRate
int8[6] inBuf
float32[6] OFC
float32[6] FSC
---
float32[6] wrench
float32[6] differential
int32[6] differential_raw
float32[6] sum
int32[6] sum_raw
float32[4] imu
float32[4] quaternion
int8[6] saturated
float32 temperature
string report_id
